algorithms/Frequent_Pattern_Mining/fpm/tg_fpm.gsql

```diff
@@ -0,0 +1,147 @@
+CREATE QUERY tg_fpm(STRING unify_vertex_type, STRING item_vertex_type, STRING connect_edge, DOUBLE support, 
+                    UINT min_length, UINT lim = 10000, INT output_limit = -1, BOOL spit_to_file = FALSE,
+                    STRING file_path = "/home/tigergraph/tg_query_output.csv") { 
+  /*
+  Schema examples: user - (buys) - product
+                   user - (visits) - page
+                   user - (rates) - product
+
+  Precondition:
+  unify_vertex_type has a List<INT> item_list as an attribute (This can be achived by running pre_process query)
+
+  Parameters:
+  unify_vertex_type - aggregate vertex
+  item_vertex_type - item vertex
+  connect_edge - undirected edge that connect the two vertices above
+
+  support: minimum frequency to be considered
+  min_length: minimum length to be considered 
+  lim: an integer to limit the length of generated sequences.
```

Comment:
The names and descriptions of min_length and lim do not sufficiently describe what they are for.

Reply:
updated